src/patternfly/components/Alert/alert.scss

```diff
@@ -128,6 +131,8 @@
 }
 
 .pf-c-alert__title {
+  display: flex;
+  align-items: center;
```

Comment:
What do we need this for? Doesn't seem necessary? If someone has `<span>`s or other elements in the title, the whitespace between them will no longer be honored.